docs/cri/containerd-config-cri-ex.x.md

```diff
@@ -0,0 +1,471 @@
+## Default CRI Configuration: Containerd 2.x:
```

Comment:
Why split from `config.md`?

Reply:
A lot of people miss content with the details buttons, after talking with some other contributors this was a reccomendation.